internal/pkg/flink/internal/controller/input_controller.go

```diff
@@ -121,6 +121,9 @@ func (c *InputController) RunInteractiveInput() {
 		})
 
 		processedStatement, err = c.store.WaitPendingStatement(ctx, *processedStatement)
+		if processedStatement != nil && processedStatement.StatusDetail != "" {
```

Comment:
But we still want to show the status detail returned from the server ;) there is a bit of refactoring to do. That's why my version added a new return value here.

Reply:
Oh okay, now I think I get it. You mean the status detail could be filled when the backend returns an error, in which case we return nil as the processed statement causing it to not be printable in the input controller and that's why you added that extra return value.

Yeah in this case it indeed needs some more refactoring. Sorry, I rushed through this a bit too quickly tbh.. I will look at it next week more thoroughly!

Reply:
Sure, some bare minimum is better than nothing. Feel free to release things incrementally. thanks!